### DISCUSSION TITLE
Trying prisma

### DISCUSSION THREAD

**Discussion by pjkennedy** [prisma/prisma#18712] [Q&A]
I tried this -

npx try-prisma@latest --template typescript/rest-nuxtjs

But it said that try-prisma wasn't available.  Can you review that it still works?  Do I need to git clone anything first? Just doing the npx script above doesn't work, so I'm missing some steps or something.

**Comment by nurul3101** [comment]
Hello @pjkennedy 👋 

I just tried the command, and it seems to be working for me

Can you check which node version is installed locally?

<img width="538" alt="Screenshot 2023-04-12 at 12 07 39 PM" src="https://user-images.githubusercontent.com/13049676/231371957-a444824c-c22d-4755-b73a-8b4737739da2.png">

**Comment by pjkennedy** [comment]
Well, that's sort of my question.  Do I need to install prisma first?

**Reply by nurul3101** [reply]
No, you shouldn't need to install prisma beforehand. try-prisma will prompt you to install all dependencies.
Can you share the exact error message that you get?

**Reply by nurul3101** [reply]
Let us know in case you are still facing this issue.

**Comment by pjkennedy** [comment]
Yes, I was trying this -

https://www.prisma.io/blog/try-prisma-announcment-Kv6bwRcdjd

npx try-prisma
Need to install the following packages:
  ***@***.***
Ok to proceed? (y)
sh: 1: try-prisma: not found

I am way out in N'jamena, Chad. However, at the U.S. Embassy, we have great
internet and a commercial grade VPN...and I can do all sorts of NPM and
yarn pulls with dozens of projects.  The only one that doesn't work
is...try-prisma.  FYI.

I will be back to Texas in late May and can try again...or I can remote
into one of my Ubuntu servers on Digital Ocean and try soon.  But, for
whatever reason, try-prisma is not working in all cases.  Any idea?  I'm
happy to troubleshoot for you, if it helps.

~Pat

**Reply by nurul3101** [reply]
Yes, it would indeed be helpful if you could help us troubleshoot this.

**Reply by nurul3101** [reply]
Let us know in case you still face this issue.

**Comment by pjkennedy** [comment]
Only another 30 days in Chad.  Will try again in U.S.  Too problematic here.

**Reply by nurul3101** [reply]
No worries, do let us know 👍

**Comment by pjkennedy** [comment]
I want to check it out. It was recommended to me by an excellent JavaScript
programmer (Chris Lis)...and it seems it will work with netlify.com per my
studies with what they support...so, I definitely will have to try it out.
I've been studying Typescript too.  But it's still a steeper hill to
climb.  I will be back in the U.S. around June 15th...so, when I get back,
I'll have to start some serious experiments and prototyping.  Just get
something working, and then expand it.  Just have to find a way to make it
work with all the JavaScript code I still have.

**Reply by nurul3101** [reply]
@pjkennedy Do let us know in case you still need any assistance.

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏

**Comment by pjkennedy** [comment]
At present, I do not need assistance.

**Comment by pjkennedy** [comment]
It works great in the States...

**Comment by pjkennedy** [comment]
RE: try-prisma issue; same error still does occur

Hi, Nurul -

I finally got back into the web dev stuff...and was working on Prisma.  I
was able to do the following tutorial perfectly:

https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgresql

...but I switched it to MySQL when it showed me how, and it was easy.  I
was able to post to the three tables--Post, User, Profile, then update
Post, etc.  It worked great.

Then I started into the next steps. But I not as sure how to set up one of
the examples. For example, where what I just tried. I cloned the git
repository -

git clone git clone https://github.com/prisma/prisma-examples.gitnpx
***@***.*** --template graphql-typegraphql-crud

And then clicked on the graphql-typegraphql-crud
<https://github.com/prisma/prisma-examples/tree/latest/typescript/graphql-typegraphql-crud>
 link.

And then I did this:

npx ***@***.*** --template graphql-typegraphql-crud

...inside the newly downloaded repository (as well as in an empty
directory). But in both cases, I got the following error:

$ npx ***@***.*** --template graphql-typegraphql-crud
sh: 1: try-prisma: not found

So, when I use the first tutorial above (which does not even use npx
***@***.***, it works), but when I try one of the prisma-examples
that does reference try-prisma, I get the error again. So, maybe I need to
set something up?  Am I missing a small step?

Thanks for any quickf pointers.

~Pat
